hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusCreateTopicHandler.java

```diff
@@ -74,7 +73,7 @@ public void preHandle(@NonNull final PreHandleContext context) throws PreCheckEx
 
         // The transaction cannot set the admin key unless the transaction was signed by that key
         if (op.hasAdminKey()) {
-            asHederaKey(op.adminKeyOrThrow()).ifPresent(context::requireKey);
+            context.requireKey(op.adminKeyOrThrow());
```

Comment:
I believe we're currently relying on `Util.asHederaKey()` to [validate this key](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/Utils.java#L38).

Reply:
yeah right. I was adding `isEmpty` method in `PreHandleContext` and use it in all places. Will probably add `isValid` as well ?

Reply:
Thanks @tinker-michaelj ! Added `isValid` method to `requireKey` method